cli-access-protected-api/README.md

```diff
@@ -0,0 +1,112 @@
+
+![A screenshot showing the output of executing the command above, which is JSON from a Graph API call.](./output.png)
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Java 11+
+- Maven
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                                    | Notes                                                                                              |
+|:------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                       | `java-cli`                                                                   | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**              | _None_                                                                       | Required value for this sample. |
+| **Client secret**              | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample with app registration values
+
+Open the [_application.properties_](src/main/resources/application.properties) file and modify the four Azure Active Directory configuration properties using the values from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app).
+
+```ini
+# Typically https://login.microsoftonline.com/tenant-id-or-primary-domain/
+authority=value-here
+
+# 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+client-id=value-here
+
+# 'Object ID' of app registration in Azure portal - this value is a GUID
+client-object-id=value-here
```

Comment:
This is used only for the Graph call, yeah? If so, note that in the comment. Better yet, use the `client-id` or another API endpoint/operation (if client-id isn't supported).

Reply:
client-id isn't supported for this call, and no other graph endpoint works without additional app configuration. This is the same as the other cli examples that we have so far.  I'll update the comment as you suggested.  Thanks!